client/onboarding/step.tsx

```diff
@@ -12,6 +11,7 @@ import { OnboardingSteps } from './types';
 import { useTrackAbandoned } from './tracking';
 import strings from './strings';
 import WooLogo from 'assets/images/woo-logo.svg';
+import CloseIcon from 'assets/images/icons/close.svg';
```

Comment:
Same comment about avoiding using SVG-in-JS imports. 🙂

Reply:
```suggestion
import CloseIcon from 'assets/images/icons/close.svg?asset';
```

Reply:
thanks @mordeth !